src/scripts/main.js

```diff
@@ -1,3 +1,66 @@
 'use strict';
 
-// write code here
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+const field = document.querySelector('.field');
+const countOfRows = field.rows;
+
+function updateButtonStates() {
+  const rows = field.rows.length;
+  const countOFCells = field.rows[0].cells.length;
```

Comment:
There is a typo in the variable name `countOFCells`. It should be `countOfCells` to maintain consistent naming conventions.